docs/deployment.md

```diff
@@ -109,21 +109,25 @@ The following services support Next.js `v12+`. Below, you’ll find examples or
 
 The following services support deploying Next.js using [`next export`](/docs/advanced-features/static-html-export.md).
 
-- [Cloudflare Pages](https://developers.cloudflare.com/pages/framework-guides/deploy-a-nextjs-site/)
-- [Firebase](https://github.com/vercel/next.js/tree/canary/examples/with-firebase-hosting)
```

Comment:
These two are not **static only** anymore so removed from here and added to their respective sections